lib/chef/provider/aws_rds_instance.rb

```diff
@@ -0,0 +1,35 @@
+require 'chef/provisioning/aws_driver/aws_provider'
+
+class Chef::Provider::AwsRdsInstance < Chef::Provisioning::AWSDriver::AWSProvider
+  REQUIRED_OPTIONS = %w(db_instance_identifier allocated_storage engine
+                        db_instance_class master_username master_user_password)
+
+  OTHER_OPTIONS = %w(engine_version multi_az iops publicly_accessible db_name db_port)
+
+  def update_aws_object(instance)
+    Chef::Log.warn("aws_rds_instance does not support modifying a started instance")
```

Comment:
Is this true?  I don't see anything about it on http://docs.aws.amazon.com/AWSRubySDK/latest/AWS/RDS/Client.html#modify_db_instance-instance_method